acquia/blt
# Account for named arguments in Twig filter and function linting.

Fixes #3178 
--------

Changes proposed:
---------
(What are you proposing we change?)

- Set Twig filters and functions added to the Twig environment BLT uses for linting to be variadic to account for named arguments.

Steps to replicate the issue:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. Create a Twig template in a custom theme template directory that includes:
`{{ drupal_entity('block', 'particle_employersmenu', check_access=false) }}`
2. Whitelist the `drupal_entity` function in blt/blt.yml per https://blt.readthedocs.io/en/latest/extending-blt/#teststwiglintall
3. Run `tests:twig:lint:all` and observe error message

Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

1. Apply this commit to BLT
2. Run issue replication steps, and confirm that linting now shows no errors.
